fixtures/pytest_store.py

```diff
@@ -207,3 +207,8 @@ def write_line(line, **kwargs):
         # resume capturing
         with diaper:
             store.capturemanager.resumecapture()
+
+
+def factory(cls, appliance_context=None, *args, **kwargs):
```

Comment:
I guess I knew that ```current_appliance``` always pointed to the "newest" appliance, but I didn't consider that assigning current_appliance to something will just assign the LocalProxy object, interesting